application/templates/application/global-html.html

```diff
@@ -13,18 +13,31 @@
           integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
     <link rel="sylesheet" type="text/css" href="{% static 'application/global-css.css' %}"> <!-- load in global css -->
     {% if title %}
-    <title>{{ title }}</title> <!-- creates title if available -->
+        <title>{{ title }}</title> <!-- creates title if available -->
     {% else %}
-    <title>Psychiatric Guide App</title> <!-- Generic Title -->
+        <title>Psychiatric Guide App</title> <!-- Generic Title -->
     {% endif %}
 </head>
 <header>
-    <center><h1>Application</h1></center>
+    <div style="text-align: center;"><h1>Application</h1></div>
```

Comment:
We should do this styling in the css files. It looks like this is used a few times, so a css class called 'div-center' or something could be created. Then if we want to update the styling of this stuff, we will just have to change it in once place

Reply:
I have this CSS done. Should get updated when we merge